src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+      class="tw-mr-12"
+    >
+      <ec-icon
+        class="ec-summary-info__main-icon"
+        :data-test="`ec-summary-info__main-icon ec-summary-info__main-icon__${iconName}`"
```

Comment:
we may delete that data-test id with iconName, because in those tests we will test Vue props passing functionality, which we shouldn't that is not our functionality.